core/templates/dev/head/player/exploration_player.html

```diff
@@ -105,6 +65,100 @@
   </script>
 
+  <script type="text/ng-template" id="popover/informationCard">
+    <md-card class="md-padding">
+      <div ng-style="infoCardBackgroundImage" class="oppia-info-card-logo-thumbnail">
+        <h2 class="oppia-info-card-title"><[explorationCardTitle]></h2>
+      </div>
+
+      <div class="oppia-info-card-content">
+        <p ng-if="explorationDescription">
+          <[explorationDescription | truncateAndCapitalize: 95]>
+        </p>
+        <p ng-if="!explorationDescription">No objective specified.</p>
+        <ul layout="row" class="card-metrics" layout-align="space-between center">
+          <li>
+            <span class="fa fa-star fa-lg"></span><[explorationRatings]></li>
+          <li>
+            <span class="fa fa-eye fa-lg"></span><[viewsCount | summarizeNumber]></li>
+          <li>
+            <span class="fa fa-clock-o fa-lg"></span><[lastModifiedDate | date: 'MM/dd/yyyy']></li>
+          <ul layout="row" layout-align="space-between center" class="oppia-info-card-exploration-contributors-profile">
+            <img src="/third_party/static/material-design-icons-1.0.1/ic_group_black_48dp.png">
+            <li ng-repeat="contributor in explorationContributors| limitTo: 2"
+              tooltip="<[contributor]>" tooltip-placement="top">
+              <img class="img-circle" ng-src="/images/general/user_mint_48px.png" alt=""></li>
+
+            <!--TODO(Makonda) get profile picture and profile url to make image click direct to contributor profile-->
+            
+            <li ng-if='explorationContributors.length > 2' class="oppia-contributors-more-circle"
+              tooltip-append-to-body=true tooltip="<[explorationContributors.slice(2).join()]>"
+              tooltip-placement="top">+<[explorationContributors.length - 2]></li>
+            <li ng-if='explorationContributors.length < 1' tooltip="public" tooltip-placement="top">
+              <span class="fa fa-globe fa-lg oppia-info-card-community-editable-icon"></span></li>
+          </ul>
+        </ul>
+        <div class="oppia-info-card-bottom-row" layout="row">
+          <div flex="55">
+            <md-content layout="row">
+              <div class="oppia-info-card-tag-icon"><span class="fa fa-tags fa-lg"></span></div>
+              <md-chips layout-align="center center">
+                  <md-chip ng-if='explorationTags.length > 2' class="oppia-info-card-tooltip-more"
+                   tooltip-append-to-body=true tooltip="<[explorationTags.slice(2).join()]>"
+                   tooltip-placement="right">
+                    <[explorationTags.length - 2]>+ more tags</md-chip>
+                  <md-chip><[explorationTags.slice(0,2).join()]></md-chip>
+                  <md-chip ng-if="explorationTags.length < 1">This exploration has no tags<md-chip>
+              </md-chips>
+            </md-content>
+          </div>
+          <div flex="45">
+            <ul layout="row" layout-align="end center">
+              <li>
```

Comment:
I spoke to Amit -- let's get rid of this `<li>` entirely, since it duplicates the embed link at the end. The G+ etc. icons should be obvious enough.